.travis.yml

```diff
@@ -1,27 +1,94 @@
 language: rust
-branches:
-  only:
-    - master
-os:
-  - linux
-  - windows
-rust:
-  - stable
-  - beta
-  - nightly
+sudo: false
+
 # Need to cache the whole `.cargo` directory to keep .crates.toml for
 # cargo-update to work, but don't cache the cargo registry
 before_cache:
   - rm -rf /home/travis/.cargo/registry
 cache:
   directories:
     - /home/travis/.cargo
-before_script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then rustup component add clippy-preview; fi
-script:
-  - if [[ "$TRAVIS_RUST_VERSION" == "stable" || "$TRAVIS_RUST_VERSION" == "nightly" ]]; then cargo clippy --all-targets --all-features -- -D warnings; fi
-  - cargo test --verbose
+
+DEPLOY_TO_GITHUB: &DEPLOY_TO_GITHUB
+  before_deploy:
+    - name="supernova-$TRAVIS_TAG-$TARGET"
+    - mkdir $name
+    - cp target/$TARGET/release/supernova $name/
+    - cp README.md LICENSE-MIT LICENSE-APACHE $name/
+    - tar czvf $name.tar.gz $name
+    - sha1sum $name.tar.gz > $name.tar.gz.sha1
+  deploy:
+    provider: releases
+    api_key: $GH_TOKEN
+    file:
+    - supernova-$TRAVIS_TAG-$TARGET.tar.gz
+    - $name.tar.gz.sha1
+    skip_cleanup: true
+    on:
+      branch: master
+      repo: SeanPrashad/supernova
```

Comment:
```suggestion
      repo: 0xazure/supernova
```

I think this slipped through during testing, but I know it was complicated flipping between projects and branches.